### PR TITLE
feat(document): make changes to text component (DEV-1147)

### PR DESCRIPTION
**Pull request by domsteinbach**
resolves DEV-1147

### CODE REVIEW

File: src/app/workspace/resource/representation/archive/archive.component.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
         <div class="toolbar">
             <!-- toolbar -->
             <div class="action horizontal bottom">
-                <!-- three dot menu to downlaod and replace file -->
+                <!-- three dot menu to download and replace file -->
```
Comment (mdelez) [inline src/app/workspace/resource/representation/archive/archive.component.html:22]: thanks 😅

File: src/app/workspace/resource/representation/text/text.component.html [hunk 2]
```diff
@@ -2,15 +2,37 @@
...
-    </button>
-    <button mat-button matTooltip="Replace text file" (click)="openReplaceFileDialog()">
-        <mat-icon>cloud_upload</mat-icon>
-    </button>
+    <div class="file-representation">
+        <div class="container">
+            <div class="contents">
+                <div class="icon">
+                    <mat-icon>
+                        description
```
Comment (mdelez) [inline src/app/workspace/resource/representation/text/text.component.html:10]: I've changed the icon for text files to "text_snippet" in #790 which should be ready for review soon. Can you change it to this preemptively please?

**Review by mdelez — CHANGES_REQUESTED** [review]

**Comment by mdelez** [comment]
also you'll need to fix the tests 🥳

File: src/app/workspace/resource/representation/text/text.component.html (hunk 2 above)
Comment (domsteinbach) [reply src/app/workspace/resource/representation/text/text.component.html]: Done.

**Comment by domsteinbach** [comment]
Changes are carried out including white content on black background menu.

**Comment by domsteinbach** [comment]
All done.

**Review by mdelez — COMMENTED** [review]
looks good, thanks

**Review by mdelez — APPROVED** [review]

**Pull request merged**